### DISCUSSION TITLE
Proposal for modifying conditional operators

### DISCUSSION THREAD

**Discussion by Yaisis** [vlang/v#17731] [Ideas]
I suggest replacing "else" and "else if" with "or" in conditional statements.
The result will be more compact and beautiful code, making the "or" operator more versatile, reducing the number of reserved words, and simplifying the language itself, since you can use one operator instead of two.

Here's an example:

``` v
if a==b {

} or {            // else

}

$if linux {

} $or windows {   // $else $if windows

} $or {           //  $else

}
```

In the example, "or" replaces "else" and "else if" - if there are no conditional statements between "or" and "{", then it replaces "else" otherwise - "else if", so you get something universal like "for"

**Comment by Delta456** [comment]
No, this sounds like a bad idea.
